### PR TITLE
feature: upgraded to 2.1.0.6 for luarocks.

### PR DESCRIPTION
**Pull request by moonming**
already uploaded the new version to luarocks: http://luarocks.org/modules/openresty/lua-cjson

### CODE REVIEW

File: lua-cjson-2.1.0.6-1.rockspec [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package = "lua-cjson"
-version = "2.1devel-1"
-
```
Comment (agentzh) [inline lua-cjson-2.1.0.6-1.rockspec]: Better keep this blank line?

Comment (moonming) [reply lua-cjson-2.1.0.6-1.rockspec]: fixed, thanks.

File: lua-cjson-2.1.0.6-1.rockspec [hunk 2]
```diff
@@ -1,8 +1,9 @@
 package = "lua-cjson"
-version = "2.1devel-1"
+version = "2.1.0.6-1"
```
Comment (agentzh) [inline lua-cjson-2.1.0.6-1.rockspec]: Why the `-1` suffix in the version number?

Comment (moonming) [reply lua-cjson-2.1.0.6-1.rockspec]: luarocks will check if the `version` follows `[%w.]+-[%d]+`, so we need the `-1` suffix here.

File: lua-cjson-2.1.0.6-1.rockspec [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
 
 source = {
-    url = "http://www.kyne.com.au/~mark/software/download/lua-cjson-2.1devel.zip",
+   url = "git+https://github.com/openresty/lua-cjson",
+   tag = "2.1.0.6",
```
Comment (agentzh) [inline lua-cjson-2.1.0.6-1.rockspec]: Why change the 4-space indentation to 3-space?

**Comment by agentzh** [comment]
Merged with fixes for the aforementioned coding style issues.

**Pull request closed without merging**